packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart

```diff
@@ -195,6 +226,61 @@ class HeapTreeViewState extends State<HeapTree> with AutoDisposeMixin {
     super.dispose();
   }
 
+  /// Detect spike in memory usage if so do an automatic snapshot.
+  void autoSnapshot({debugMode = false}) {
+    final heapSample = controller.memoryTimeline.sampleAddedNotifier.value;
+    final heapSum = heapSample.external + heapSample.used;
+    heapMA.addToMA(heapSum);
+
+    final DateFormat mFormat = DateFormat('hh:mm:ss.mmm');
```

Comment:
avoid abbreviations. `mFormat`--> `timeFormat`? I'm not sure what the `m` stands for.

Reply:
dateTimeFormat